sql-odbc/docs/dev/sign_installers.md

```diff
@@ -6,7 +6,7 @@
   - Buy [Extended Validation (EV) Code Signing Certificate](https://docs.microsoft.com/en-us/windows-hardware/drivers/dashboard/get-a-code-signing-certificate#step-2-buy-a-new-code-signing-certificate)
   - Create a [self-signed certificate](https://docs.microsoft.com/en-us/windows/msix/package/create-certificate-package-signing#create-a-self-signed-certificate) (For testing purpose only).
 - Install the certificate that you want to sign the file with.
-  - [Import-Certificate](https://docs.microsoft.com/en-us/powershell/module/pkiclient/import-certificate?view=win10-ps) can be used for this purpose
+
```

Comment:
probably better to update the instruction? instead of removing it
https://docs.microsoft.com/en-us/powershell/module/pki/import-certificate?view=windowsserver2019-ps

Reply:
I could not find an updated link for this, think this should be safe to remove since it is all internal-facing, please confirm @penghuo

Reply:
I posted the updated link in my comment? I believe it's just the doc for `Import-Certificate` powershell command, url says `sindowsserver2019-ps` but this also applies to win10-ps

Reply:
Whoops, i thought you had just copy-pasted the original link- sure, can replace

Reply:
We are good to remove the instruction, customer should not sign it.